[Fix] p6spy 삭제 및 like엔티티 -> likes로 변경

## 🤔 오류 내용
에러로그 함께 입력  
<br>
Caused by: java.io.FileNotFoundException: class path resource [com/github/gavlyukovskiy/boot/jdbc/decorator/DataSourceDecoratorAutoConfiguration.class] cannot be opened because it does not exist

어제까지 잘만되던 p6spy가 잘 안돌아가서 그냥 삭제했습니다. 문제가 뭔지 모르겠는데 중요한 기능은 아니라서 제거했을 때 문제가 없으니 제거한 상태로 진행하려고 합니다.

mysql 예약어 중에 like가 있어서 우리가 만든 like엔티티랑 충돌이 나서 likes로 변경합니다.
@Table(schema = "likes")이렇게 하면 테이블 이름 likes로 바껴서 괜찮을 줄 알았는데 아니였습니다. (테이블 잘 만들어 지는거 확인 꼭 하기!)

## ⚠ 에러 캡쳐 

<br>

--- a/src/main/java/com/bada/Backend/domain/likes/entity/likes.java
+++ b/src/main/java/com/bada/Backend/domain/likes/entity/likes.java
@@ -6,7 +6,7 @@ import lombok.Getter;
 @Entity
 @Getter
 @Table(schema = "likes")
-public class like {
+public class likes {
     @Id
     @GeneratedValue
     @Column(name = "like_id")

--- a/src/main/java/com/bada/Backend/domain/likes/repository/LikesRepository.java
+++ b/src/main/java/com/bada/Backend/domain/likes/repository/LikesRepository.java
@@ -1,8 +1,8 @@
 package com.bada.Backend.domain.likes.repository;
 
-import com.bada.Backend.domain.likes.entity.like;
+import com.bada.Backend.domain.likes.entity.likes;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 
-public interface LikesRepository extends JpaRepository<like,Long> {
+public interface LikesRepository extends JpaRepository<likes,Long> {
 }

--- a/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports
+++ b/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports
@@ -1,1 +1,0 @@
-com.github.gavlyukovskiy.boot.jdbc.decorator.DataSourceDecoratorAutoConfiguration

--- a/src/main/resources/spy.properties
+++ b/src/main/resources/spy.properties
@@ -1,1 +1,0 @@
-appender=com.p6spy.engine.spy.appender.Slf4JLogger

